e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/OrderTest.groovy

```diff
@@ -0,0 +1,65 @@
+package gov.hhs.cdc.trustedintermediary.e2e
+
+import spock.lang.Specification
+
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class OrderTest extends Specification {
+
+    def orderClient = new EndpointClient("/v1/etor/orders")
+    def labOrderJsonFileString = Files.readString(Paths.get("src/test/resources/lab_order.json"))
+
+    def "an order response is returned from the ETOR order endpoint"() {
+        given:
+        def expectedFhirResourceId  = "Bundle/969bcbb3-cd34-49be-ac4f-e1b8479b8219"
+        def expectedPatientId  = "MRN7465737865"
+
+        when:
+        def responseBody = orderClient.submit(labOrderJsonFileString)
+        def parsedJsonBody = JsonParsing.parse(responseBody, Map.class)
+
+        then:
+        parsedJsonBody.fhirResourceId == expectedFhirResourceId
+        parsedJsonBody.patientId == expectedPatientId
+    }
+
+    def "bad response given for poorly formatted JSON"() {
+
+        when:
+        def responseBody = orderClient.submit(labOrderJsonFileString.substring(1))
+        //removed beginning '{' to make this JSON invalid
+
+        then:
+        responseBody == "Server Error"
+    }
+
+    def "payload file check"() {
```

Comment:
I think a better test of the sent payload is only check that it is completely the same as the input FHIR.

Reply:
Do you mean something like this? 
![Screenshot 2023-07-05 at 11 50 52 AM](https://github.com/CDCgov/trusted-intermediary/assets/86614374/904e7aed-c1f9-46ec-b34a-461daab103ea)

Reply:
Yes, exactly.